core/src/main/java/com/klaviyo/coresdk/KlaviyoConfig.kt

```diff
@@ -0,0 +1,34 @@
+package com.klaviyo.coresdk
+
+data class KlaviyoConfig (
+        val apiKey: String,
+        val networkTimeout: Int,
+        val networkFlushInterval: Int
+    ) {
+    class Builder {
+        private val NETWORK_TIMEOUT_DEFAULT: Int = 500
+        private val NETWORK_FLUSH_INTERVAL_DEFAULT: Int = 60000
+
+        private lateinit var apiKey: String
+        private var networkTimeout: Int = NETWORK_TIMEOUT_DEFAULT
+        private var networkFlushInterval: Int = NETWORK_FLUSH_INTERVAL_DEFAULT
```

Comment:
should we validate inputs?

Reply:
What kind of validation are you thinking? Ensuring numbers are always positive? Throwing exceptions or just reverting to defaults on invalid types?
Given that these values will have healthy defaults in place, I'm thinking of just using the default if the received value is invalid and logging a warning for the user rather than outright throwing an exception.

Reply:
yeah validation would really just be: are the numbers > 0, i like the idea of defaulting if its not the case and logging a warning to the dev. wondering how we log things, if a dev does use a bad value, how do we feel about this logging getting onto the customer's device (if they ship it anyway with the bad value)?

Reply:
Ah that's a good callout, luckily something I already documented in the RFCs but forgot to stroymap a ticket out for so I'll do that soon. But basically we can configure Timber to plant a debug tree so it only logs on debug builds, not release builds.